.github/workflows/nightly-tests.yml

```diff
@@ -7,12 +7,23 @@ jobs:
   nightly-test:
     name: Nightly test
     runs-on: ubuntu-latest
-    container: kedacore/build-tools:v2
```

Comment:
operator-sdk is already installed on this image, so if we run on this, we can skip thi installation step below